okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/ConjureWindowedLimit.java

```diff
@@ -38,7 +38,7 @@
     /**
      * Minimum observed samples to filter out sample windows with not enough significant samples.
      */
-    private static final int WINDOW_SIZE = 100;
+    private static final int WINDOW_SIZE = 10;
```

Comment:
stays in sync with the netflix class: https://github.com/Netflix/concurrency-limits/commit/b108aa802a98086117a95afb52e43c865a8c4160